### PR TITLE
docs(en-US): tutorial for traditional web

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add a tutorial post: Integrate with traditional web app.

## Background

We don't have any traditional web app SDK for v1, so I created a "smallest" express sample: https://github.com/logto-io/express-example, it is written by pure JavaScript for Node and doesn't have lint configured, keep it simple in order that users can follow this tutorial and understand how to integrate Logto to a traditional web app, especially when they are using other framworks or tech stacks.

### CODE REVIEW

File: docs/docs/recipes/integrate-logto/traditional.mdx [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+```bash
+pnpm add @logto/js node-fetch@v2 express-session js-base64
+```
+
+</TabItem>
+</Tabs>
+
+## Use session
+
+When a user is signed in, it will get a set of tokens(access token, id token, refresh token) and interaction data, session is a good place to store them.
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/traditional.mdx:68]: better add whitespace before the left parenthesis

File: docs/docs/recipes/integrate-logto/traditional.mdx [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+
+:::tip
+
+This artical is not just for Express, if you are using other frameworks, or even other languages, you can replace `@logto/js` with other language's core SDK, and then adjust some of the steps.
+
+:::
+
+## Start an Express project
+
+With `express-generator`, you can easily start an express project.
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/traditional.mdx:25]: express → Express

File: docs/docs/recipes/integrate-logto/traditional.mdx [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+    requester
+  );
+
+  return tokenResponse;
+};
+```
+
+## Sign in
+
+Create a route in express to sign in:
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/traditional.mdx:198]: express → Express

File: docs/docs/recipes/integrate-logto/traditional.mdx [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+```js
+app.get('/sign-out', (req, res) => {
+  req.session.tokens = null;
+  res.send('Sign out successful');
+});
+```
+
+## Protect resource
+
+Create a middleware named `withAuth` to attach an `auth` object to `req`, and verify if a user is signed in.
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/traditional.mdx:249]: "verify if a user is signed in" → "verify if a user signed in"

File: docs/docs/recipes/integrate-logto/traditional.mdx [hunk 5]
```diff
@@ -0,0 +1,334 @@
...
+      req.auth = req.session.tokens.idToken.sub;
+    }
+
+    next();
+  };
+
+module.exports = withAuth;
+```
+
+create `index` page, show a sign in link for guests, and a go to profile link for users that already signed in:
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/traditional.mdx:291]: Improved by Grammarly:

create `index` page, show a sign-in link for guests, and a go-to profile link for users that have already signed in

File: docs/docs/recipes/integrate-logto/traditional.mdx [hunk 6]
```diff
@@ -0,0 +1,334 @@
...
+
+```jade
+extends layout
+
+block content
+  h1 Hello logto
+  p userId: #{userId}
+```
+
+## Get Source Code
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/traditional.mdx:326]: according to the discussion, we use sentence case on the title.

should be "Get source code"

File: docs/docs/recipes/integrate-logto/traditional.mdx [hunk 7]
```diff
@@ -0,0 +1,334 @@
...
+
+You can go to [GitHub](https://github.com/logto-io/express-samle) to get the final code for this tutorial.
+
+## Further Readings
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/traditional.mdx:330]: ditto. should be "Further readings"

File: docs/docs/recipes/integrate-logto/traditional.mdx [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+cd express-logto
+npx express-generator
+```
+
+## Install dependencies
+
+The demo app will need 3 dependencies:
+
+1. **@logto/js**: Logto's core SDK for JavaScript.
+2. **node-fetch**: Minimal code for a window.fetch compatible API on Node.js runtime.
```
Comment (gao-sun) [inline docs/docs/recipes/integrate-logto/traditional.mdx:38]:
```suggestion
2. **node-fetch**: Minimal code for a `window.fetch` compatible API on Node.js runtime.
```

**Review by gao-sun — APPROVED** [review]
approve to unblock - let's refine this with grammarly later

**Review by gao-sun — CHANGES_REQUESTED** [review]
since we're using grammarly, let's refine it in this PR

File: docs/docs/recipes/integrate-logto/traditional.mdx (hunk 4 above)
Comment (wangsijie) [reply docs/docs/recipes/integrate-logto/traditional.mdx:249]: Looks odd for me.

**Comment by wangsijie** [comment]
Refined with grammarly, PTAL @gao-sun

**Review by IceHe — APPROVED** [review]
Overall LGTM

**Review by gao-sun — APPROVED** [review]

File: docs/docs/recipes/integrate-logto/traditional.mdx [hunk 9]
```diff
@@ -0,0 +1,334 @@
...
+extends layout
+
+block content
+  h1 Hello logto
+  p userId: #{userId}
+```
+
+## Get source code
+
+You can go to [GitHub](https://github.com/logto-io/express-samle) to get the final code for this tutorial.
```
Comment (IceHe) [inline docs/docs/recipes/integrate-logto/traditional.mdx:328]: real link is https://github.com/logto-io/express-example
```suggestion
You can go to [GitHub](https://github.com/logto-io/express-example) to get the final code for this tutorial.
```

**Pull request merged**